src/patternfly/components/Toolbar/toolbar.scss

```diff
@@ -110,6 +110,15 @@ $pf-v6--include-toolbar-breakpoints: true !default;
   row-gap: var(--#{$toolbar}__content--RowGap);
   padding-inline-start: var(--#{$toolbar}__content--PaddingInlineStart);
   padding-inline-end: var(--#{$toolbar}__content--PaddingInlineEnd);
+
+  &.pf-m-toolbar-filtered-items {
+    display: flex;
+    align-items: baseline;
+
+    > :first-child {
```

Comment:
I'm wondering if we'd ever expect more than one toolbar group? Then they wouldn't get spaced correctly.
<img width="873" alt="image" src="https://github.com/user-attachments/assets/6e3d675d-5375-4e73-b036-3459a038680c" />

Also, can you add this new class to the documentation section?